### PR TITLE
basic-starter-react tweaks

### PR DESCRIPTION
**Pull request by iangilman**

### CODE REVIEW

**Comment by iangilman** [comment]
@Daniel15 just a few tweaks on top of your fine example. I'm new to JSX, so let me know if I did something dumb.

File: basic-starter-react/index.html [hunk 1]
```diff
@@ -20,8 +20,8 @@
           loggedIn: R.authenticated(),
           name: R.authenticated() ? R.currentUser.get('firstName') : null,
           playState: undefined,
-          topTrack: undefined,
```
Comment (Daniel15) [inline basic-starter-react/index.html:23]: Thanks! At work we have a transform to remove trailing commas (for IE support) so I've gotten into the habit of always using them :P

**Comment by Daniel15** [comment]
Your tweaks all look good :)
Out of curiosity, why aren't the examples on the Rdio site?

**Comment by iangilman** [comment]
It was just easier to put them on my site, and then we never got around to moving them over. Perhaps someday :-)

Thanks for the review, and thanks again for the patch!

**Pull request merged**